frontend/web/components/MetadataSelect.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, useState } from 'react'
+import InlineModal from './InlineModal'
+import Input from './base/forms/Input'
+import Utils from 'common/utils/utils'
+import Icon from './Icon'
+
+export type MetadataSelectType = {
+  disabled: boolean
+  contentType: number
+  metadata: [] | undefined
+  value: number[] | undefined
+  isOpen: boolean
+  onAdd: (id: number) => void
+  onRemove: (id: number) => void
+  onToggle: () => void
+}
+const MetadataSelect: FC<MetadataSelectType> = ({
+  disabled,
+  isOpen,
+  metadata,
+  onAdd,
+  onRemove,
+  onToggle,
+  value,
+}) => {
+  const [filter, setFilter] = useState<string>('')
+  const metadataList =
+    metadata &&
+    metadata.filter((v) => {
+      const search = filter.toLowerCase()
+      if (!search) return true
+      return `${v.name}`.toLowerCase().includes(search)
+    })
+  return (
+    <InlineModal
+      title='Metadata'
+      isOpen={isOpen}
+      onClose={onToggle}
+      className='inline-modal--tags'
```

Comment:
Can this be replaced with inline-modal--sm

Reply:
Done